assets/model_monitoring/components/src/data_quality_statistics/compute_data_quality_statistics.py

```diff
@@ -133,8 +153,11 @@ def compute_data_quality_statistics(df):
     """Compute data quality statistics."""
     dtype_df = get_df_schema(df=df).to_pandas_on_spark()
     unique_vals_df = get_unique_value_list(df=df)
-    max_vals = compute_max_df(df=df.to_pandas_on_spark())
-    min_vals = compute_min_df(df=df.to_pandas_on_spark())
+    # Note: excluding boolean type column as the boolean type do not need to be calculated
```

Comment:
Is boolean the only type doesn't support max/min value operation here?

Reply:
As I added supported type list, we support 6 datatypes in pyspark: [StringType, IntegerType, DoubleType, ByteType, LongType, FloatType (float might not be accurate, we suggest double instead)]. For not supported type, we just do not hav max, min value and we just do not generate the outofbands metrics. We do not fail the job or throw exceptions for unsupported types.